lib/vkloader.c

```diff
@@ -749,46 +749,62 @@ linearTilingPadCallback(int miplevel, int face,
  * limited number of formats and features. Generally @c VK_IMAGE_TILING_OPTIMAL
  * is preferred. The latter requires a staging buffer so will use more memory
  * during loading.
+ * 
+ * If a pointer to a set of suballocator callbacks is provided, they
+ * will be used instead of manual allocation of VkDeviceMemory. A 64 bit uint
+ * that references the suballocated page(s) is returned on memory procurement 
+ * and saved in the @c allocationId field of the structure pointed to by @A vkTexture.
  *
- * @param[in] This          pointer to the ktxTexture from which to upload.
- * @param [in] vdi          pointer to a ktxVulkanDeviceInfo structure providing
- *                          information about the Vulkan device onto which to
- *                          load the texture.
- * @param [in,out] vkTexture pointer to a ktxVulkanTexture structure into which
- *                           the function writes information about the created
- *                           VkImage.
- * @param [in] tiling       type of tiling to use in the destination image
- *                          on the Vulkan device.
- * @param [in] usageFlags   a set of VkImageUsageFlags bits indicating the
- *                          intended usage of the destination image.
- * @param [in] finalLayout  a VkImageLayout value indicating the desired
- *                          final layout of the created image.
+ * @param[in] This                        pointer to the ktxTexture from which to upload.
+ * @param [in] vdi                        pointer to a ktxVulkanDeviceInfo structure providing
+ *                                        information about the Vulkan device onto which to
+ *                                        load the texture.
+ * @param [in,out] vkTexture              pointer to a ktxVulkanTexture structure into which
+ *                                        the function writes information about the created
+ *                                        VkImage.
+ * @param [in] tiling                     type of tiling to use in the destination image
+ *                                        on the Vulkan device.
+ * @param [in] usageFlags                 a set of VkImageUsageFlags bits indicating the
+ *                                        intended usage of the destination image.
+ * @param [in] finalLayout                a VkImageLayout value indicating the desired
+ *                                        final layout of the created image.
+ * @param [in] subAllocatorCallbacks      pointer to a set of suballocator callbacks 
+ *                                        that wrap around suballocator calls: alloc, 
+ *                                        bindbuffer, bindimage, map, unmap and free.
+ *                                        They use a uint64_t stored in the @c allocationId
+ *                                        field of the structure pointed at by @A vkTexture
```

Comment:
s/@A/@a/. We're victims of autocorrect here too.

Reply:
Fixed!